express-intro/server.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log(`The date is ${new Date()}`);
+  next();
+});
+
+app.listen(8000, () => {
```

Comment:
Remember to use 8080 in all future exercises, instead of 8000. This will become important as we build upon what we are learning now.

Reply:
Noted! Misread the instruction.